server/src/main/java/org/opensearch/action/search/DeletePitResponse.java

```diff
@@ -57,7 +56,6 @@ public List<DeletePitInfo> getDeletePitResults() {
      */
     @Override
     public RestStatus status() {
-        if (deletePitResults.isEmpty()) return NOT_FOUND;
```

Comment:
So what would be the payload returned from the method? Empty list of PITs?

Reply:
The server returns 200 StatusCode with { "pits": [] } as the body when there are no PITs to delete.